torcharrow/velox_rt/dataframe_cpu.py

```diff
@@ -308,6 +308,9 @@ def append(self, values: Iterable[Union[None, dict, tuple]]):
             return self
 
     def _check_columns(self, columns: Iterable[str]):
+        if isinstance(columns, str):
```

Comment:
should we just check `columns` should be a `Iterable[str]`? -- basically enforce the type annotation?

Reply:
any advice to check it? ```isinstance(columns, Iterable[str])``` doesnt work